sdk/eventhub/event-hubs/src/checkpointManager.ts

```diff
@@ -40,7 +40,8 @@ export interface Checkpoint {
 }
 
 /**
- * CheckPointManager is created by the library & passed to user's code to let them create a checkpoint
+ * The checkpoint manager that clients should use to update checkpoints to track progress of events processed. Each
```

Comment:
"clients" can be confusing here... which client? Am guessing we mean users/customers

```suggestion
 * The checkpoint manager is used to update checkpoints to track progress of events processed. Each
```